project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.7
\ No newline at end of file
+sbt.version=1.3.6
```

Comment:
was there something which wasn't supported or something?

Reply:
Intellij can't import projects with sbt > 1.3.6. After some investigation it seems to be a bug with Coursier which has been raised. One way around it is to import the project using version 1.3.6 and once imported bump the version back up to 1.3.8